app/src/main/res/values/strings.xml

```diff
@@ -176,6 +176,7 @@
     <string name="pref_keep_screen_on_title">Keep Screen On</string>
     <string name="pref_keep_screen_on_summary">Prevent the screen from turning off whilst Clementine Remote is connected to Clementine.
     </string>
+    <string name="pref_noti_color_title">Turn the color notifications</string>
```

Comment:
Better name it `Use system notification background color` and add a summary with more information about this.